src/app/server/Mdns.h

```diff
@@ -19,26 +19,37 @@
 
 #include <core/CHIPError.h>
 #include <stddef.h>
+#include <support/BitFlags.h>
 
 namespace chip {
 namespace app {
 namespace Mdns {
 
+enum class CommissioningMode
+{
+    kDisabled,
+    kEnabled,
+    kEnabledAsAdditionalCommissioning
```

Comment:
It's worth documenting when one would use which of these; `kEnabledAsAdditionalCommissioning` means that we are in commissioning mode as a result of the "Open Commissioning Window" command or "Open Basic Commissioning Window" command and must not be set in any other cases, right?

Reply:
added comments